Package build fails with signing error

**What happened**:

Running `make -C packages` results in a build error:

```
 dpkg-genchanges  >../dracut_051-1.1_amd64.changes
dpkg-genchanges: info: not including original source code in upload
 dpkg-source --after-build .
dpkg-source: info: using options from dracut-051/debian/source/options: --extend-diff-ignore=(^|/).+(\.asc)$
dpkg-buildpackage: info: binary and diff upload (original source NOT included)
Now signing changes and any dsc files...
 signfile dsc dracut_051-1.1.dsc Garden Linux Maintainers <contact@gardenlinux.io>
gpg: keybox '/home/dev/.gnupg/pubring.kbx' created
gpg: skipped "Garden Linux Maintainers <contact@gardenlinux.io>": No secret key
gpg: /tmp/debsign.5iWbRKXq/dracut_051-1.1.dsc: clear-sign failed: No secret key
debsign: gpg error occurred!  Aborting....
debuild: fatal error at line 1112:
running debsign failed
make: *** [Makefile:4: dracut] Error 29
make: *** [Makefile:44: pipeline] Fehler 2
make: Verzeichnis „/home/mba/Projekte/SAP/git/gardenlinux/packages“ wird verlassen
```

**What you expected to happen**:

Package get signed correctly, and building proceeds for next package

**How to reproduce it (as minimally and precisely as possible)**:

Run `make -C packages` in the top-level directory, or `make` in the `packages/` directory

**Environment**:

Debian stable

--- a/packages/Makefile
+++ b/packages/Makefile
@@ -54,7 +54,7 @@ pipeline: docker sign
 		--env BUILDIMAGE=$(BUILDIMAGE) \
 		--env DEBFULLNAME=$(DEBFULLNAME) \
 		--env DEBEMAIL=$(DEBEMAIL) \
-		$(BUILDIMAGE) make
+		$(BUILDIMAGE) bash -c "gpg --import /sign.pub; make"
 manual-kernel: docker-kernel
 	docker run --rm -ti --volume "$(POOLDIR)":/pool --volume "$(KERNELDIR)":/home/dev/manual \
 	       	--env BUILDTARGET="$(POOLDIR)" \

